docs/contributor/arch/021-refactor-performance-tests.md

```diff
@@ -0,0 +1,44 @@
+# 20. Refactor Performance Tests
+
+Date: 2025-07-03
+
+## Context
+
+Currently, [performance tests](../benchmarks/README.md) are written in bash which makes them hard to maintain. Our goal is to rewrite them in Golang so that they become easier to read, update and debug.
+
+### POC for using components from Opentelemtry Collector Testbed
+
+Since we are going to rewrite the performance tests in Golang, this is a good point of time to check if it is usefule to use components from the [Opentelemetry Collector Testbed](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/testbed).
```

Comment:
```suggestion
Because we are going to rewrite the performance tests in Golang, this is a good point of time to check if it is useful to use components from the [Opentelemetry Collector Testbed](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/testbed).
```